backend/taxHandlers.ts

```diff
@@ -67,42 +67,47 @@ const calculateTaxes = async (
   taxData: FetchTaxesPayload,
   taxJarConfig: TaxJarConfig
 ): Promise<{ data: ResponseTaxPayload }> => {
-  const taxResposne = await fetchTaxes(taxData, taxJarConfig);
+  const linesWithChargeTaxes = taxData.lines.filter(
+    (line) => line.chargeTaxes === true
+  );
+  const taxDataWithChargeTaxes = {
+    ...taxData,
+    lines: linesWithChargeTaxes,
+  };
+  const taxResposne =
+    linesWithChargeTaxes.length !== 0
+      ? await fetchTaxes(taxDataWithChargeTaxes, taxJarConfig)
+      : undefined;
 
-  const taxDetails = taxResposne.tax.breakdown;
+  const taxDetails = taxResposne?.tax.breakdown;
```

Comment:
If `taxResponse` will be `undefined` you have to do this, so the app won't crash:

```suggestion
  const taxDetails = taxResposne?.tax?.breakdown;
```

Reply:
You don't @witoszekdev. As long as `.tax` is not optional, `taxResposne?.tax.breakdown` is fine and will never crash.